routes/map.js

```diff
@@ -9,9 +10,12 @@ const router = express.Router();
 
 router.get(
   "/:id/points",
+  validateId,
   validateToken,
   validateMember,
   mapController.getMapPoints
 );
 
+router.post("/new", validateToken, mapController.createNewMap);
```

Comment:
```suggestion
router.get(
  "/:id/points",
  validateId,
  validateToken,
  validateMember,
  mapController.getMapPoints
);
router.post("/new", validateToken, mapController.createNewMap);
```

이 부분은 붙이는게 좋을까요 떼는게 좋을까요..?

Reply:
그러고보니 저도 한줄로 작성될때 붙여서 작성했었던 것 같아요!  
성훈님 의견은 어떠신가요??

Reply:
저는 개행 시켜주는 것이 좋아 보이는 것 같은데 다른 분들은 어떻게 생각하실까요??

Reply:
현재는 관련성이 높은 것들끼리 묶어두고,  나중에 리팩토링 할 때 정해봐요!